pki.spec

```diff
@@ -155,7 +155,7 @@ BuildRequires:    policycoreutils
 
 # Java build dependencies
 BuildRequires:    %{java_devel}
-BuildRequires:    maven-local-openjdk17
```

Comment:
Which is the jdk installed in this case?

Reply:
The `java_devel` is still defined as `java-17-openjdk-devel`, so it should install OpenJDK 17.

The Maven dependency was originally defined as `maven-local`, which works fine on Fedora 39, but on Fedora 40 it seems to be pulling OpenJDK 21 in addition to OpenJDK 17. I was trying to complete Fedora 40 support first and then add OpenJDK 21 support separately later, so I changed the dependency to `maven-local-openjdk17` specifically, but apparently this package is not available on CentOS/RHEL, so for now it's reverted back to `maven-local`.